avocado/utils/cloudinit.py

```diff
@@ -174,6 +174,22 @@ def __init__(self, address, instance_id):
         self.instance_id = instance_id
         self.instance_phoned_back = False
 
+    def wait_for_phone_home(self, new_call=False):
+        """Waits for this instance to call.
+
+        :param new_call: Should we ignore previous calls and
+                         wait for the next call? Or should we consider previous
+                         calls as well?  Default is False, so if this instance
+                         it was called back already this method it will return
+                         immediately.
```

Comment:
Hi @beraldoleal the code LGTM. But I don't understand this docstring. I would remove those questions and just describe what the `new call` is doing, what do you think? Also, I think the last sentence is not grammatically correct. I think it should be:
 ```
Default is False, so if the instance was called back already, this method will return immediately.
```
But my English isn't perfect, so I might be wrong.

Reply:
I can remove the questions, and I will fix the sentence, but I would like to ensure we are talking about "this instance" instead of "the instance".